### ISSUE TITLE
Support for multiple configurations

### ISSUE THREAD

**Issue by postmechanical** [modcloth-labs/github-xcode-bot-builder#2]
It doesn't appear as this tool supports more than one configuration. I work on multiple projects for which I'd like to use this tool. Each project, however, has a unique configuration.

**Comment by geoffnix** [comment]
Yes, this currently only supports one configuration.  This would be a great feature to add.  We aren't likely to have time to add this in the near future.  If you do add support for this, please send us a pull request!
